packages/siwx/src/core/SIWXMessenger.ts

```diff
@@ -1,17 +1,42 @@
 import type { SIWXMessage } from '@reown/appkit-core'
 
+/**
+ * This is the base class for a SIWX messenger.
+ * It provides the basic structure for creating a SIWX message and util functions.
+ */
 export abstract class SIWXMessenger {
   public domain: SIWXMessage['domain']
   public uri: SIWXMessage['uri']
   public statement?: SIWXMessage['statement']
   public resources?: SIWXMessage['resources']
 
+  /**
+   * The version of the SIWXMessage class
+   */
```

Comment:
what's the current / defualt version?

Reply:
there is no default version, this is an abstract class, the one extending it must provide a version